src/raw/requests.rs

```diff
@@ -268,10 +265,8 @@ impl KvRequest for kvrpcpb::RawBatchDeleteRequest {
 
     fn map_result(_: Self::RpcResponse) -> Self::Result {}
 
-    fn reduce(
-        results: BoxStream<'static, Result<Self::Result>>,
-    ) -> BoxFuture<'static, Result<Self::Result>> {
-        results.try_collect().boxed()
+    async fn reduce(results: BoxStream<'static, Result<Self::Result>>) -> Result<Self::Result> {
+        results.try_collect().await
```

Comment:
Is it better to use `try_for_each_concurrent` instead of `try_collect`?

Reply:
`try_for_each_concurrent` only returns `()`, so I don't think we can use it instead of `try_collect`.

Reply:
But the result types are all `()` where we use `try_collect`

Reply:
Oh, in which case we probably can!